consts.php

```diff
@@ -107,12 +107,131 @@
 const INSTALLER_TO_REPO_MINOR_VERSIONS = [
     '4.10' => [
-        'silverstripe-html5' => '2.3',
+        'agency-extensions' => '2.6',
+        'comment-notifications' => '2.2',
+        'cwp' => '2.9',
+        'cwp-core' => '2.9',
+        'cwp-pdfexport' => '1.3',
+        'cwp-search' => '1.6',
+        'silverstripe-advancedworkflow' => '5.6',
+        'silverstripe-akismet' => '4.2',
+        'silverstripe-auditor' => '2.4',
+        'silverstripe-blog' => '3.9',
+        'silverstripe-ckan-registry' => '1.4',
+        'silverstripe-comments' => '3.7',
+        'silverstripe-composer-update-checker' => '2.1',
+        'silverstripe-config' => '1.3',
+        'silverstripe-content-widget' => '2.3',
+        'silverstripe-contentreview' => '4.4',
+        'silverstripe-crontask' => '2.4',
+        'silverstripe-documentconverter' => '2.2',
+        'silverstripe-elemental' => '4.8',
         'silverstripe-elemental-bannerblock' => '2.4',
+        'silverstripe-elemental-fileblock' => '2.3',
+        'silverstripe-elemental-userforms' => '3.1',
+        'silverstripe-environmentcheck' => '2.4',
+        'silverstripe-externallinks' => '2.2',
+        'silverstripe-fluent' => '4.6',
+        'silverstripe-fulltextsearch' => '3.9',
+        'silverstripe-graphql' => '3.7',
+        'silverstripe-gridfieldqueuedexport' => '2.6',
+        'silverstripe-html5' => ['2.2', '2.3'],
+        'silverstripe-hybridsessions' => '2.4',
+        'silverstripe-iframe' => '2.2',
+        'silverstripe-ldap' => '1.3',
+        'silverstripe-login-forms' => '4.6',
+        'silverstripe-maintenance' => '2.4',
+        'silverstripe-mfa' => '4.5',
+        'silverstripe-mimevalidator' => '2.3',
+        'silverstripe-multivaluefield' => '5.2',
+        'silverstripe-queuedjobs' => '4.9',
+        'silverstripe-realme' => '4.2',
+        'silverstripe-registry' => '2.4',
+        'silverstripe-restfulserver' => '2.4',
+        'silverstripe-security-extensions' => '4.2',
+        'silverstripe-securityreport' => '2.4',
+        'silverstripe-segment-field' => '2.5',
         'silverstripe-session-manager' => '1.2',
-        'silverstripe-userforms' => '5.12',
+        'silverstripe-sharedraftcontent' => '2.6',
+        'silverstripe-sitewidecontent-report' => '3.2',
+        'silverstripe-spamprotection' => '3.2',
+        'silverstripe-spellcheck' => '2.3',
+        'silverstripe-subsites' => '2.5',
+        'silverstripe-tagfield' => '2.8',
+        'silverstripe-taxonomy' => '2.3',
+        'silverstripe-textextraction' => '3.3',
         'silverstripe-totp-authenticator' => '4.3',
+        'silverstripe-userforms' => ['5.11', '5.12'],
+        'silverstripe-versionfeed' => '2.2',
+        'silverstripe-webauthn-authenticator' => '4.4',
+        'silverstripe-widgets' => '2.2',
+        'starter-theme' => '3.2',
+        'watea-theme' => '3.1',
+    ],
+    '4.11' => [
+        'agency-extensions' => '2.7',
```

Comment:
actually `cwp-agencyextensions` in github (https://github.com/silverstripe/cwp-agencyextensions)
```suggestion
        'cwp-agency-extensions' => '2.7',
```